### PR TITLE
Version 5

### LINKED ISSUE #277: Interceptor not called in lazy-loading module with imported GalleryModule
#### What is the current behavior?
When I importing the GalleryModule in lazy-loading module my interceptor stops working in this module. I suppose that happens because I use the HttpClientModule in AppModule and in GalleryModule also used HttpClientModule. But I should import HttpClientModule only once, see [docs](https://angular.io/guide/http#setup-installing-the-module)

What can I do to resolve this problem?

#### Which versions are you using for the following packages?

Angular: 7.0.0
Angular CDK: 7.1.0
Angular CLI: 7.0.2
Typescript: 3.1.1
Gallery: 4.0.2-beta.0

### LINKED ISSUE #321: <gallery> component  unable to render YouTube video consistently in Angular 9
#### What is the expected behaviour?

<gallery> component can render YouTube video without a TypeError in Angular 9

I.e. this.galleryRef.addYoutube({...}) 

#### What is the current behaviour?

TypeError: Cannot read property 'nativeElement' of undefined.

The stackblitz below shows the video unable to render at all in the component. I do however have the video rendering and functioning outside of stackblitz, the TypeError is still present though.

#### What are the steps to reproduce?

https://stackblitz.com/edit/ngx-gallery-g5bijs

#### Which versions are you using for the following packages?

Angular: 9.0.1
Angular CDK: 9.0.0
Angular CLI: 9.0.1
Typescript: 3.7.4
Gallery: 4.0.3

#### Is there anything else we should know?

Starting with version 9, the resolution strategy for @ViewChild and @ContentChild queries defaults to false , which means query results will be available in ngAfterViewInit unless specified otherwise (i.e. {static: true}).

https://angular.io/guide/static-query-migration

The ngx-gallery core code in the file gallery-iframe.component.ts looks to be affected by this change.

https://github.com/MurhafSousli/ngx-gallery/blob/8dea5fc1bfbe9767824e0b9e69601e794ed7a5e7/projects/core/src/lib/components/templates/gallery-iframe.component.ts#L29

and

https://github.com/MurhafSousli/ngx-gallery/blob/8dea5fc1bfbe9767824e0b9e69601e794ed7a5e7/projects/core/src/lib/components/templates/gallery-iframe.component.ts#L22

### PR DESCRIPTION
**Pull request by MurhafSousli**
## BETA DEMO https://ngx-gallery.netlify.app/#/

It will take me sometime until I update the documentation and the demo.

Please install the package using `npm i ng-gallery`

## Changelog of v5

- Upgrade to Angular 9, closes #321.
- Combine all packages in one package.
- Remove import `HttpClientModule` from `GalleryModule`, closes #277.

### Breaking changes

**Before:**

- The packages were published on `@ngx-gallery/core`, `@ngx-gallery/lightbox` and `@ngx-gallery/gallerize`.

**After:**

- All the packages are now combined in `ng-gallery` (NOTE: it is not `ngx-gallery` that is a different package).
- Import `GalleryMlodule` from `ng-gallery` and `LightboxModule` from `ng-gallery/lightbox`.
- The module `GallerizeModule` has been removed, the `[gallerize]` directive can still be used from the `LightboxModule`.

### CODE REVIEW

**Comment by MurhafSousli** [comment]
https://stackblitz.com/edit/ngx-gallery updated to v5.0.0-beta.2

**Comment by kamilxx22** [comment]
@MurhafSousli 
Hi,
Can I know when you are switching to the latest version?
I care about changes in HttpClientModule.

Best wishes
Kamil

**Comment by duongle26** [comment]
Please update the doc and release v5

**Comment by MurhafSousli** [comment]
@duongle26 will do that when I have time

For core-js you should not update it to the latest version (this is not related to this plugin but to angular framework), so stick to core-js version used in the stackblitz

File: projects/ng-gallery-demo/src/app/shared/badges/badges.component.html [hunk 1]
```diff
@@ -14,13 +14,13 @@
   <img data-canonical-src="https://travis-ci.org/MurhafSousli/ngx-gallery"
        src="https://travis-ci.org/MurhafSousli/ngx-gallery.svg?branch=master">
 </a>
-<a href="https://www.npmjs.com/package/@ngx-gallery/core">
-  <img data-canonical-src="https://www.npmjs.com/package/@ngx-gallery/core"
-       src="https://img.shields.io/npm/dt/@ngx-gallery/core.svg?maxAge=2592000?style=plastic">
+<a href="https://www.npmjs.com/package/ngx-gallery">
```
Comment (matheo) [inline projects/ng-gallery-demo/src/app/shared/badges/badges.component.html:17]: @MurhafSousli isn't the `ngx-gallery` package a different one?

**Comment by matheo** [comment]
@MurhafSousli do you think v5.0.0 can be used in an Angular 8 application?

File: projects/ng-gallery-demo/src/app/pages/documentation/doc-core/doc-core/doc-core.component.ts [hunk 2]
```diff
@@ -8,23 +8,23 @@ import { ChangeDetectionStrategy, Component } from '@angular/core';
 })
 export class DocCoreComponent {
 
-  docs = `import { GalleryModule } from  '@ngx-gallery/core';
+  docs = `import { GalleryModule } from  'ngx-gallery';
```
Comment (matheo) [inline projects/ng-gallery-demo/src/app/pages/documentation/doc-core/doc-core/doc-core.component.ts:11]: `ngx-gallery`?

File: projects/ng-gallery-demo/src/app/shared/badges/badges.component.html (hunk 1 above)
Comment (MurhafSousli) [reply projects/ng-gallery-demo/src/app/shared/badges/badges.component.html:17]: Oh yea, thanks for noticing! will need to fix that

**Pull request merged**